README.md

```diff
@@ -1,20 +1,18 @@
-# qbicsoftware/rnadeseq
+# qbic-pipelines/rnadeseq
```

Comment:
If you like to, could also add the title image here as in bamtofastq or remove it there, to have a similar look. However, I don't think it is as important to have for our organisation

Reply:
yeah why not, it looks really cool in your pipeline 😄